content/nap-waf/v5/configuration-guide/configuration.md

```diff
@@ -791,15 +791,42 @@ To enable mTLS in NGINX, you need to perform the following steps:
     
 ## Brute Force Attack Preventions
 
-Brute force attacks are attempts to break in to secured areas of a web application by trying exhaustive, 
+### Overview
+
+Brute force attacks are attempts to break in to secured areas of a web application by trying exhaustive,
 systematic, username/password combinations to discover legitimate authentication credentials. 
-To prevent brute force attacks, WAF tracks the number of failed attempts to reach login pages 
-with enforced brute force protection. When brute force patterns are detected, 
-the WAF policy considers it to be an attack if the failed logon rate increased significantly or 
-if failed logins reached a maximum threshold.
+To prevent brute force attacks,  NGINX App Protect WAF monitors both IP addresses and/or usernames and tracks the number of 
+failed login attempts reached a maximum threshold. 
```

Comment:
```suggestion
To prevent brute force attacks, NGINX App Protect WAF monitors IP addresses, usernames, and the number of failed login attempts beyond a maximum threshold. 
```